packages/cli-template-monorepo-subgraph/apps/contracts/package.json

```diff
@@ -1,42 +1,54 @@
 {
     "name": "monorepo-subgraph-contracts",
-    "version": "1.0.0",
-    "private": true,
-    "main": "index.js",
     "scripts": {
-        "dev": "hardhat node & yarn compile && yarn deploy --network localhost",
+        "dev": "hardhat node & yarn deploy --network localhost",
         "compile": "hardhat compile",
         "deploy": "yarn compile && hardhat deploy",
+        "verify": "hardhat verify",
         "test": "hardhat test",
         "test:report-gas": "REPORT_GAS=true hardhat test",
         "test:coverage": "hardhat coverage",
         "typechain": "hardhat typechain",
-        "lint": "solhint 'contracts/**/*.sol'"
+        "lint": "eslint . --ext .js,.ts && solhint 'contracts/**/*.sol'",
+        "prettier": "prettier -c --plugin=prettier-plugin-solidity .",
+        "prettier:write": "prettier -w --plugin=prettier-plugin-solidity ."
     },
     "devDependencies": {
-        "@nomicfoundation/hardhat-chai-matchers": "^2.0.3",
+        "@nomicfoundation/hardhat-chai-matchers": "^2.0.0",
         "@nomicfoundation/hardhat-ethers": "^3.0.0",
         "@nomicfoundation/hardhat-network-helpers": "^1.0.0",
         "@nomicfoundation/hardhat-toolbox": "^4.0.0",
         "@nomicfoundation/hardhat-verify": "^2.0.0",
-        "@semaphore-protocol/core": "4.0.0-alpha.8",
-        "@semaphore-protocol/hardhat": "4.0.0-alpha.8",
+        "@semaphore-protocol/core": "^4.0.0-alpha.8",
+        "@semaphore-protocol/hardhat": "^4.0.0-alpha.8",
         "@typechain/ethers-v6": "^0.5.0",
         "@typechain/hardhat": "^9.0.0",
         "@types/chai": "^4.2.0",
-        "@types/mocha": "^9.1.1",
+        "@types/eslint": "^8",
+        "@types/mocha": ">=9.1.0",
+        "@types/node": "^20.11.20",
+        "@typescript-eslint/eslint-plugin": "^7.0.2",
+        "@typescript-eslint/parser": "^7.0.2",
         "chai": "^4.2.0",
-        "dotenv": "^14.3.2",
+        "dotenv": "^16.4.5",
+        "eslint": "^8.57.0",
+        "eslint-config-airbnb-base": "^15.0.0",
+        "eslint-config-airbnb-typescript": "^17.1.0",
+        "eslint-config-prettier": "^9.1.0",
+        "eslint-plugin-import": "^2.29.1",
         "ethers": "^6.4.0",
-        "hardhat": "^2.19.4",
+        "hardhat": "^2.20.1",
         "hardhat-gas-reporter": "^1.0.8",
+        "prettier": "^3.2.5",
         "prettier-plugin-solidity": "^1.3.1",
-        "solhint": "^3.3.6",
-        "solhint-plugin-prettier": "^0.0.5",
-        "solidity-coverage": "^0.8.0",
-        "typechain": "^8.3.0"
+        "solhint": "^4.1.1",
+        "solidity-coverage": "^0.8.1",
+        "ts-node": "^10.9.2",
+        "typechain": "^8.3.0",
+        "typescript": "^5.3.3"
     },
     "dependencies": {
-        "@semaphore-protocol/contracts": "4.0.0-alpha.8"
-    }
+        "@semaphore-protocol/contracts": "^4.0.0-alpha.8"
```

Comment:
It would be better to not use `^` and specify the version here. `^` may introduce unexpected issues as it could use different versions.